src/meta/app/src/schema/table.rs

```diff
@@ -250,9 +251,23 @@ impl TableMeta {
         Ok(())
     }
 
+    pub fn add_column(
+        &mut self,
+        field: &TableField,
+        comment: &str,
+        index: FieldIndex,
+    ) -> Result<()> {
+        let mut new_schema = self.schema.as_ref().to_owned();
+        new_schema.add_column(field, index)?;
+        self.schema = Arc::new(new_schema);
+        self.field_comments.insert(index, comment.to_owned());
+        Ok(())
+    }
+
     pub fn drop_column(&mut self, column: &str) -> Result<()> {
```

Comment:
why change `drop_column` return type, it seems that there is no place to use this API this pr.

Reply:
> drop_column

Because we need use the fieldIndex to remove the comment in `field_comments`

Reply:
index is used to remove `field_comments`, which is not handled before.